packages/election/graphql/resolvers/Candidacy.js

```diff
@@ -0,0 +1,27 @@
+const { Roles, transformUser } = require('@orbiting/backend-modules-auth')
+
+module.exports = {
+  user: (candidacy, args, { user: me }) => transformUser(candidacy.user),
+  yearOfBirth: (candidacy, args, { user: me }) => {
+    if (!Roles.userIsInRoles(me, ['admin', 'supporter', 'associate'])) {
+      return ''
+    }
+
+    if (!candidacy.user.birthday) {
+      return
```

Comment:
will make boom, because `undefined` is not an `int!`

Reply:
make it `int` in schema, this shouldn't happen but eventually one user might need/want to delete their birthday